frontend/src/components/notifications/notificationcard/NotificationCardController.jsx

```diff
@@ -1,24 +1,29 @@
+import moment from 'moment'
```

Comment:
```suggestion
// eslint-disable-next-line import/no-unresolved
import moment from 'moment'
```

Reply:
hmm, seems like i have no perms to accept this suggested, change. @arathibiju can u merge the change on github by clicking the accept change button etc.

Reply:
yeah it's a shitty issue with github actions, think rebasing ur branch should solve it, but i can't be bothered